tests/renderer/investtab.test.js

```diff
@@ -179,6 +217,28 @@ describe('Save InVEST Model Setup Buttons', () => {
     expect(spy).toHaveBeenCalledTimes(0);
     spy.mockRestore(); // restores to unmocked implementation
   });
+
+  test('Load Parameters Button: does nothing when canceled', async () => {
+    // this resembles the callback data if the dialog is canceled instead of 
+    // a save file selected.
+    const mockDialogData = {
+      filePaths: ['']
+    };
+    ipcRenderer.invoke.mockResolvedValue(mockDialogData);

```

Comment:
Neat! I'm guessing we need this because it's not possible to test things in the choose-a-file window?

Reply:
That's right!